runner/bin/run/check.js

```diff
@@ -6,22 +7,26 @@ const FORM_PATH = path.join(process.cwd(), "src", "server", "forms");
 
 async function check() {
```

Comment:
I cant see unit test for this function. Is there one already?

Reply:
I've written the tests but I've had to refactor/move around functions so that Sinon can stub it properly. IMO it's kinda ugly but should have a higher confidence.

I'm not entirely sure if I can test `check` itself, it's a void function and is quite low level.

Reply:
I tried to get sinon to spy on cli and process but no dice. hopefully these two tests are enough